pkg/catalogv2/oci/client.go

```diff
@@ -262,7 +262,7 @@ func (o *Client) GetOrasRegistry() (*remote.Registry, error) {
 	}
 
 	orasRegistry.Client = &auth.Client{
-		Cache: auth.DefaultCache,
+		Cache: nil,
```

Comment:
Please add a comment to explain why caching should be avoided specially across OCI repo objects.